Readme-LavaInstallation.md

```diff
@@ -224,12 +236,24 @@ cat <<EOT >> /etc/samba/smb.conf
    guest ok = yes
 EOT
 service smbd restart
+```
+To verify if the SMB share is working type this:
+```bash
 mount //$(hostname)/lava /var/lib/lava/dispatcher/tmp
 # press Enter (as there is an empty password)
 ```
 
+Go to http://your-lava-machine-ip/admin/lava_scheduler_app/device/localhost/ and don't forget to change your-lava-machine-ip into an actual ip.
+Change the Device type into hyperv.
+
+```bash
+cp https://github.com/ader1990/lava-dispatcher/blob/hyperv_booting_method/binaries/ssh-retry.sh /root/lava-dispatcher/ssh.sh
+```
+
+
 ### Run instructions
 To be added
+Type into an web browser the IP of your LAVA machine, login with the credentials that you typed into Configuration instructions, go to Scheduler, Submit Job, type in the text area the script from this link http://paste.ubuntu.com/25719905/, Validate and Submit. If all is working well you will have no errors.
```

Comment:
* Go to http://<LAVA-IP>/, login with the superuser credentials
* Go to  http://<LAVA-IP>/scheduler/jobsubmit and use http://paste.ubuntu.com/25719905 as an example job definition and press `Submit`.